com.unity.multiplayer.mlapi/Runtime/Transports/NetworkTransport.cs

```diff
@@ -19,6 +19,11 @@ public enum NetworkChannel : byte
         NetworkVariable, //todo: this channel will be used for snapshotting and should then go from reliable to unreliable
         SnapshotExchange,
         Fragmented,
+        ReliableSequenced,
+        ReliableUnsequenced,
+        ReliableFragmentedSequenced,
+        UnreliableSequenced,
+        UnreliableUnsequenced,
```

Comment:
FWIW, I believe we're planning to get rid of the explicit transport QoS channel specification as a whole soon :)

Reply:
Yes, really soon :)